Source/EasyNetQ.Management.Client/ManagementClient.cs

```diff
@@ -84,15 +90,19 @@ static ManagementClient()
         string password,
         TimeSpan? timeout = null,
         Action<HttpRequestMessage>? configureHttpRequestMessage = null,
-        Action<HttpClientHandler>? configureHttpClientHandler = null
+        Action<HttpHandler>? configureHttpHandler = null
     )
     {
         if (!endpoint.IsAbsoluteUri) throw new ArgumentOutOfRangeException(nameof(endpoint), endpoint, "Endpoint should be absolute");
 
         this.configureHttpRequestMessage = configureHttpRequestMessage;
 
-        var httpHandler = new HttpClientHandler { Credentials = new NetworkCredential(username, password) };
-        configureHttpClientHandler?.Invoke(httpHandler);
+        # if NET6_0
```

Comment:
if/else branches are identical 😕

Reply:
Lol, thanks

Reply:
Sorry, I was thinking about branching here, but then decided to conditionally import a type.